data/gegno/ships.txt

```diff
@@ -0,0 +1,770 @@
+
+ship Protolith
+	sprite ship/protolith
+	thumbnail thumbnail/protolith
+	swizzle 0
+	attributes
+		category "Heavy Freighter"
+		cost 17600000
+		mass 1260
+		bunks 292
+		"cargo space" 530
+		drag 16.1
+		"engine capacity" 180
+		"fuel capacity" 700
+		"heat dissipation" 0.4
+		hull 19575
+		"outfit space" 768
+		"required crew" 84
+		shields 0
+		"weapon capacity" 300
+		weapon
+			"blast radius" 280
+			"shield damage" 1550
+			"hull damage" 2050
+			"hit force" 3950
+	outfits
+		"C27 Campaign Core"
+		"Chloeric Turret" 2
+		Hyperdrive
+		"Irate Cannon" 2
+		"R02 Battlezone Battery"
+		"R03 Warforge Battery"
+		"RG3 Torch Steering"
+		"RG3 Torch Thruster"
+	engine 0 250
+	engine -41.5 242
+	engine 41.5 242
+	gun 37 -212 "Irate Cannon"
+	gun 45 -199.5 "Irate Cannon"
+	turret 0 -33 "Chloeric Turret"
+	turret 0 8 "Chloeric Turret"
+	leak leak 30 50
+	leak flame 30 80
+	leak "big leak" 50 30
+	explode "huge explosion" 30
+	explode "large explosion" 45
+	explode "medium explosion" 35
+	explode "small explosion" 25
+	"final explode" "final explosion large"
+	description "Protoliths are descendents of the Gegno's early city ships used to colonize the nearby planets. They carry hundreds of battle-ready Vi both in and out of war, and carry huge cargo loads to supply depots. They are not typically used as warships but can be mounted with a small payload that can threaten most ships smaller than them, or fend off larger ships long enough for reinforcements to arrive."
+
+ship Halite
+	sprite ship/halite
+	thumbnail thumbnail/halite
+	swizzle 0
+	attributes
+		category Fighter
+		cost 129000
+		mass 30
+		bunks 1
+		"cargo space" 1
+		drag 0.74
+		"engine capacity" 30
+		"heat dissipation" 1
+		hull 860
+		"outfit space" 120
+		"required crew" 1
+		shields 0
+		"weapon capacity" 30
+		weapon
+			"blast radius" 14
+			"shield damage" 160
+			"hull damage" 85
+			"hit force" 160
+	outfits
+		"SC-1 Plasma Engines"
+		"I70 Ameliorate Cell"
+		"Talc Laser" 2
+		
+	engine -9 42
+	engine 9 42
+	gun -18.5 -44.5 "Talc Laser"
+	gun 18.5 -44.5 "Talc Laser"
+	leak leak 60 50
+	explode "small explosion" 8
+	explode "tiny explosion" 10
+	description "Halites are a double-first for the Gegno. They are the Gegno Scin's newest ship design, and are the Gegno's first fighter-based craft. They are also the first Gegno ships equipped with early-stage beam weapon technology, with almost half the hull dedicated to it. Able to fly around the Gegno Vi's slower projectile techology, Halites are able to bring a swarm of instant damage to the slow-moving Vi warships, making them hard to deal with in numbers."
```

Comment:
What do you mean by the `almost half the hull dedicated to it`? I wouldn't think that the hull mattered much when considering what guns are equipped.

Reply:
It's more of a visual design reference than an actual game mechanic reference- the model of the Halite has it so that the two gun mounts are in fact the lasers themselves tied into the frame, and the Talc Laser in the outfitter is what you see on the Halite itself.

Reply:
Maybe this would make it clearer?
```suggestion
	description "Halites are a double-first for the Gegno. They are the Gegno Scin's newest ship design, and are the Gegno's first fighter-based craft. They are also the first Gegno ships equipped with early-stage beam weapon technology, with almost half the hull dedicated to holding two Talc Lasers. Able to fly around the Gegno Vi's slower projectile techology, Halites are able to bring a swarm of instant damage to the slow-moving Vi warships, making them hard to deal with in numbers."
```